fhservices.cfg.j2

```diff
@@ -121,6 +121,7 @@ define service {
 
 define service {
        service_description Storage::Pod::Disk Usage
+       # REVIEW: is this being applied to every pod/container? How does it work?
        hosts {{ rhmap_router_dns }}
```

Comment:
@jessesarn I see this was added in a previous PR, but just wondered should we be using 'hostgroup_name' here instead of 'hosts' like all the other services?

```
hostgroup_name core,mbaas
```

It will work like this since it's supposed to be applied in all cases, but might be best to keep them all consistent